module/User/src/Entity/User.php

```diff
@@ -1,60 +1,81 @@
 <?php
-
 namespace User\Entity;
 
 use DateTime;
 use ZfcRbac\Identity\IdentityInterface;
 
 class User implements IdentityInterface
 {
+
```

Comment:
In this project uses PSR-4 code style. For fixing code style you need to use command `composer fix` that will running `php-cs-fixer` command tool.